cs3api4lab/utils/file_utils.py

```diff
@@ -37,3 +37,18 @@ def calculate_content_size(content):
             content_len = len(content.decode('utf-8'))
         content_size = str(content_len)
         return content_size
+
+    @staticmethod
+    def normalize_path(path):
+        if len(path) > 0 and path[0] != '/':
+            path = '/' + path
+        elif path == '' or path is None:
+            path = '/'
+        return path
+
+    @staticmethod
+    def remove_drives_names(path):
+        drives = ["cs3drive:", "cs3driveShareWithMe:", "cs3driveShareByMe:"]
+        for drive in drives:
```

Comment:
for drive in ["cs3drive:", "cs3driveShareWithMe:", "cs3driveShareByMe:"]:
Moze taki zapis byly lepszy? to juz zalezy od zespolu jak chcemy sie umowic na zapis takich rzeczy